### PR TITLE
Sanitizes adminhelp

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
You can no longer ahelp blankly
Ahelping is a pop-up now instead of text to type to the message, you can't spam it using a macro anymore for example

:cl:
 * bugfix: Adminhelping has more sanity.

### CODE REVIEW

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -15,22 +15,15 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
...
-		return
-
-	/**src.verbs -= /client/verb/adminhelp
-
-	spawn(1200)
-		src.verbs += /client/verb/adminhelp	// 2 minute cool-down for adminhelps
-		src.verbs += /client/verb/adminhelp	// 2 minute cool-down for adminhelps//Go to hell
-	**/
-
+	var/msg = input(usr, "What message would you like to adminhelp?", "Send a message to admins", "")
```
Comment (N3X15) [inline code/modules/admin/verbs/adminhelp.dm]: This is worded awkwardly.

File: code/modules/admin/verbs/adminhelp.dm [hunk 2]
```diff
@@ -15,22 +15,15 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
...
 	//clean the input msg
 	if(!msg)
 		return
-	msg = sanitize(copytext(msg,1,MAX_MESSAGE_LEN))
+	msg = trim_left(sanitize(copytext(msg,1,MAX_MESSAGE_LEN)))
```
Comment (N3X15) [inline code/modules/admin/verbs/adminhelp.dm]: Should trim right as well.

**Review by N3X15 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/adminhelp.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/modules/admin/verbs/adminhelp.dm]: Give me a better one

File: code/modules/admin/verbs/adminhelp.dm (hunk 2 above)
Comment (boy2mantwicethefam) [reply code/modules/admin/verbs/adminhelp.dm]: Will fix

File: code/modules/admin/verbs/adminhelp.dm (hunk 1 above)
Comment (N3X15) [reply code/modules/admin/verbs/adminhelp.dm]: * "What do you need help with?"
* "What message would you like to send to the administrators?"
* "What is the nature of your emergency?"

Comment (boy2mantwicethefam) [reply code/modules/admin/verbs/adminhelp.dm]: I'll go for 2, thanks

**Review by N3X15 — APPROVED** [review]

**Pull request merged**